tests/test_username_api.py

```diff
@@ -1,24 +1,29 @@
-import unittest
 import json
-import yaml
-import string
+import logging
 import random
+import string
+import os.path
```

Comment:
as you are moving imports around, please move them into alpha order per PEP8

Reply:
sure

Reply:
done